packages/atomic/cypress/integration/breadbrumb-manager.cypress.ts

```diff
@@ -24,7 +23,7 @@ const categoryFacetProp = {
   label: 'Atlas',
 };
 
-describe.skip('Breadcrumb Manager Test Suites', () => {
+describe('Breadcrumb Manager Test Suites', () => {
```

Comment:
I assume there were skipped because they started failing. I fixed that, see below.